lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -167,3 +184,13 @@ def test_missing_thread_id(self):
     def test_missing_endorsed(self):
         self.form_data.pop("endorsed")
         self.assert_field_value("endorsed", None)
+
+    @ddt.data("True", "true", 1, True)
+    def test_endorsed_true(self, value):
```

Comment:
Same question for `endorsed` as for `following`.
Although not part of the ticket, since we're looking at the test coverage again, is there a `test_invalid_endorsed` method?  That is, is there a unit test for when someone passes an invalid value for `endorsed`?